docs/source/metoffice.rst

```diff
@@ -0,0 +1,165 @@
+.. _MetOffice:
+
+Using Fab at the Met Office
+===========================
+
+For users working on Met Office desktop systems the following instructions may
+prove useful.
+
+Using Fab
+~~~~~~~~~
+
+If you just want to use Fab to build some source then there are a number of
+fairly simple approaches.
+
+
+Fortran Only
+------------
+
+If you don't need to build C, you can use this the modules in the "extra
+software" collection:
+
+.. code-block:: console
+
+    $ module use $EXTRA_SOFTWARE
+    $ module load python/3.7.0 support/fparser/0.16.0/python/3.7.0 fab/0.11.0/python/3.7.0
+
+The Science I.T. group can provide the `EXTRA_SOFTWARE` path.
+
+
+Fortran and C
+-------------
+
+If you need to build C, or you use conda environments anyway you can use the
+:ref:`Conda<Anaconda>` route to Fab-ulousness.
+
+You will need to authenticate with Artifactory the first time you use Conda.
+There is information regarding this on MetNet or ask Science I.T.
+
+
+.. _Run Singularity:
+
+Singularity Container
+---------------------
+
+You can run Fab in a singularity container as follows::
+
+    singularity run oras://$URL/metomi/fab/MyImage:latest
+
+The necessary URL is that normally used with singularity. Ask Science I.T. for
+details.
+
+If you need to use git from within the container, you'll need to set a couple of environment variables first::
+
+    $ export SINGULARITY_BIND="/etc/pki/ca-trust/extracted/pem:/pem"
+    $ export SINGULARITYENV_GIT_SSL_CAPATH="/pem"
+    $ singularity shell oras://$URL/fab/MyImage:latest
+
+See also :ref:`Instructions for building the image<Build Singularity>`.
+
+
+Developing Fab
+~~~~~~~~~~~~~~
+
+If you want to develop Fab then there are some additional steps which may be
+needed.
+
+
+Mixing Virtual Environment and Environment Modules
+--------------------------------------------------
+
+If you prefer to use the built-in Python virtual environment tool for
+development you can combine it with the "extra software" modules. The
+complication comes because they already make use of such an environment.
+
+First load the environment you want to use to compile your software. The LFRic
+environment is used as an example.
+
+.. code-block:: console
+
+    $ module use $EXTRA_SOFTWARE
+    $ module use $LFRIC_SOFTWARE
+    $ module load environment/lfric/ifort
+
+Then create a new virtual environment, unload the existing one and activate
+the new.
+
+.. code-block:: console
+
+    $ python3 -m venv $VENV_DIR/fab_dev
+    $ module unload python/3.7.0
+    $ source $VENV_DIR/fab_dev/bin/activate
+
+Install all the bits needed to install PSyclone and other parts of the build
+process.
+
+.. code-block:: console
+
+    $ pip install --upgrade pip
+    $ pip install Jinja2
+    $ pip install six
+    $ pip install fparser
+    $ pip install pyparsing
+    $ pip install sympy
+    $ pip install jsonschema==3.0.2
+    $ pip install configparser
+
+Finally install the Fab in your working copy. This is done in "editable" mode
```

Comment:
Very minor comment, but should "the" be removed from here? So:
```suggestion
Finally, install Fab in your working copy. This is done in "editable" mode
```

Reply:
Done